.trivyignore

```diff
@@ -1,2 +1,6 @@
 # ignore protobuf cve temporary
 CVE-2024-7254
+# ignore commons-io-2.11.0.jar introduced by odps-reader
```

Comment:
shall submit a separate PR for this CVE change.

Reply:
> shall submit a separate PR for this CVE change.

chat with smitch-chen, he will deal with these cve together